template/interpolate/aws/secretsmanager/types.go

```diff
@@ -0,0 +1,36 @@
+package secretsmanager
+
+import (
+	"github.com/aws/aws-sdk-go/service/secretsmanager/secretsmanageriface"
+)
+
+// AWSConfig store configuration used to initialize
+// secrets manager client.
+type AWSConfig struct {
+	Region string
+}
+
+// SecretSpec represent specs of secret to be searched
+// If Key field is not set then package will return first
+// secret key stored in secret name.
+//
+// maps to ClusterConfig
+type SecretSpec struct {
+	Name string
+	Key  string
+}
+
+// Client represents an AWS Secrets Manager client
+//
+// maps to ProviderServices
+type Client struct {
```

Comment:
I think this type can in the client file https://github.com/hashicorp/packer/pull/9202/files#diff-a6b302ac91e064234f987dc6032905a5

Reply:
Do you mean move `Client` struct to `secretsmanager.go` file?

Reply:
Do you mean move `Client` strcut to `secretsmanager.go` file?

Reply:
Oops, yes. Sorry about the confusion.